src/engine/source/cmds/src/test.cpp

```diff
@@ -18,43 +31,80 @@ constexpr auto OUTPUT_ONLY {0};
 constexpr auto OUTPUT_AND_TRACES {1};
 constexpr auto OUTPUT_AND_TRACES_WITH_DETAILS {2};
 
-void run(std::shared_ptr<apiclnt::Client> client, const Parameters& parameters)
+int clear_icanon(bool flag)
```

Comment:
Can we use a more descriptive name for the flag? can we return an `int32_t` or `uint32_t` type here?

Reply:
Or a bool